src/workloads/execution/MultiDeletesDocByDoc.yml

```diff
@@ -0,0 +1,27 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Runs a workload that inserts and then deletes a large range of documents on one collection while
+  performing writes on another collection in the same database. Serves as a base line for comparing
+  legacy document by document deletes versus efficiently batched multi deletes.
+
+# Increase the maxPoolSize to enable faster initial data population
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 300
+      # Allow for longer duration since the loading of the data may take a while.
+      socketTimeoutMS: 3_600_000  # = 1 hour
```

Comment:
The deletes right now take about 7 minutes. This is to prevent timeouts with minimal information in case something gets really broken and a delete takes forever